packages/fuels-code-gen/src/program_bindings/abigen/bindings/contract.rs

```diff
@@ -40,7 +40,8 @@ pub(crate) fn contract_bindings(
         pub struct #name<T: ::fuels::accounts::Account> {
```

Comment:
```suggestion
        #[derive(Debug, Clone)]
        pub struct #name<T: ::fuels::accounts::Account> {
```
IMO we should add `Clone` so that the user can clone the instance. See comments below.